tv_apps/smart-system/js/app_install_manager.js

```diff
@@ -292,6 +322,72 @@ var AppInstallManager = {
     app.onprogress = this.handleProgress;
   },
 
+  previewApp: function ai_previewApp(app) {
+    if (!this.isMarketPlaceAppActive()) {
+      return;
+    }
+
+    var marketplace = AppWindowManager.getActiveApp();
+    var manifestURL = app.manifestURL;
+    var appURL = app.origin +
+      (app.manifest.launch_path === '/' ? '/' : app.manifest.launch_path);
+
+    marketplace.childWindowFactory.createPreviewWindow({
+      detail: {
+        url: appURL,
+        origin: app.origin,
+        manifestURL: manifestURL
```

Comment:
Since the variable `manifestURL` will be used only once, how about use `app.manifestURL` directly?